File: mobi_client/mobi_client/Assets/scripts/Config/ConfigModel/RoleConfigModel.cs
namespace ConfigSystem
{
﻿    public class RoleConfigModel :ConfigModel {
        public string name;//名字
        public int appear;//是否显示
        public string alias;//别名
        public string english;//英文名称
        public string nickname;//讯息昵称
        public string icon;//头像
        public string modelName;//模型名称
        public string drawing;//立绘
        public int fight;//加入战斗需要的羁绊等级
        public int heroid;//对应的heroid
        public int recordid;//对应的档案id
        public string site1;//看板位置1
        public string site2;//看板位置2
        public System.Collections.Generic.List<string> standbyAct;//主界面待机动作
        public string leisureAct;//主界面休闲动作
        public System.Collections.Generic.List<System.Collections.Generic.List<string>> leisureTalk;//主界面休闲对话
        public string interactAct;//主界面互动动作
        public System.Collections.Generic.List<System.Collections.Generic.List<string>> interactTalk;//主界面互动对话
        public System.Collections.Generic.List<string> talkAvg;//交流avg
    }
}

File: mobi_client/mobi_client/mobi_config/excel_check/temporary/java/RoleConfigModel.java
package com.mobi.config.configModel;
import com.mobi.config.ConfigModel;
import com.mobi.config.wrappedArrayList.*;
public class RoleConfigModel extends ConfigModel {
	public String name;//名字
	public int appear;//是否显示
	public String alias;//别名
	public String english;//英文名称
	public String nickname;//讯息昵称
	public String icon;//头像
	public String modelName;//模型名称
	public String drawing;//立绘
	public int fight;//加入战斗需要的羁绊等级
	public int heroid;//对应的heroid
	public int recordid;//对应的档案id
	public String site1;//看板位置1
	public String site2;//看板位置2
	public ArrString standbyAct;//主界面待机动作
	public String leisureAct;//主界面休闲动作
	public ArrArrString leisureTalk;//主界面休闲对话
	public String interactAct;//主界面互动动作
	public ArrArrString interactTalk;//主界面互动对话
	public ArrString talkAvg;//交流avg
}